aries-site/src/examples/foundation/icons/IconComponentExample.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Box, Button, Menu } from 'grommet';
+import { Add, FormNext } from 'grommet-icons';
+
+export const IconComponentExample = () => {
+  return (
+    <Box gap="medium" direction="row-responsive">
+      <Menu icon={<Add />} label="Add User" />
```

Comment:
I think if we are going to use a Menu example, we should have items in it so when the user clicks, it shows some items